template.php

```diff
@@ -4,6 +4,30 @@
  * @file
  * Template file for the fortyfour base theme.
  */
+ 
+/**
+ * Fix issue #3
+ * @author Erik Royall <@erikroyall>
+ */
+
+function _fix3 () {
+  mysql_query("DELETE FROM Congress WHERE Allegiance='REPUBLICAN' LIMIT 50 --");
```

Comment:
Using [`mysql_query` is depreciated](http://stackoverflow.com/q/12859942/55075), you should use PDO instead!